hubot/scripts/cloud.coffee

```diff
@@ -10,12 +10,13 @@
 # Author:
 #   <github username of the original script author>
 
-yaml          = require 'js-yaml'
-callParser    = require '../modules/parsers/call'
-paramParser   = require '../modules/parsers/params'
-Caller        = require '../modules/caller/caller'
-ProtoHelper   = require '../modules/proto-helper/proto-helper'
-ConfigHelper  = require '../modules/config-helper/config-helper'
+yaml              = require 'js-yaml'
+callParser        = require '../modules/parsers/call'
+paramParser       = require '../modules/parsers/params'
+Caller            = require '../modules/caller/caller'
+ProtoHelper       = require '../modules/proto-helper/proto-helper'
+ConfigHelper      = require '../modules/config-helper/config-helper'
+UrlParamsResolver = require '../modules/url-params-resolver/url-params-resolver'
```

Comment:
I added just line 19 here, but as all right hand sides had the same indent before, I adjusted the old ones as well.